package.json

```diff
@@ -4,7 +4,7 @@
   "description": "chat",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "mocha --compilers js:babel-core/register --require ./app/tests/test_helper.js './app/tests/**/*.@(js|jsx)'",
```

Comment:
It's better to rename it to `test-client`, as there also will be `test-server`.
Also it's better to not depend on global presence of mocha and use local version of it: `./node_modules/mocha/bin/mocha`